PixiEditor/Helpers/Converters/NotNullToVisibilityConverter.cs

```diff
@@ -6,13 +6,14 @@
 namespace PixiEditor.Helpers.Converters
 {
     [ValueConversion(typeof(object), typeof(Visibility))]
-    public class NotNullToVisibilityConverter : IValueConverter
+    public class NotNullToVisibilityConverter
+        : MarkupConverter
     {
         public bool Inverted { get; set; }
 
-        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        public override object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
-            bool isNull = value != null;
+            bool isNull = value is not null;
```

Comment:
how is this more readable than previous one

Reply:
I once read that this is a safer way of null check since you can change how `!=` but not how `is not` works

Reply:
Fair point